inst/pages/11_taxonomic_information.qmd

```diff
@@ -120,6 +120,21 @@ head(getUniqueFeatures(tse, rank = "Phylum"))
 
 ### Generate a hierarchy tree on the fly {#sec-fly-tree}
 
+A hierarchy tree is not an actual phylogenic tree.
+A phylogenic tree represents evolutionary relationships among organisms while
+a hierarchy tree organizes species into a hierarchical structure based on their 
+taxonomic ranks.
```

Comment:
in fact the taxonomic ranks also reflect evolutionary relationships.. you can say that it shows mapping between the taxonomic levels in taxonomic rank table (included in rowData), rather than the detailed phylogenetic relations included in phylogenetic tree.